language/ui-patterns/patterns/elements/index.md

```diff
@@ -0,0 +1,8 @@
+---
+type: ??
```

Comment:
Does it matter that we'll have 3 of the same type of page? The Elements, Components, and Layouts pages will all be this same type of list page.